setup.py

```diff
@@ -96,7 +96,7 @@
 		'bin/fail2ban-client',
 		'bin/fail2ban-server',
 		'bin/fail2ban-regex',
-		'bin/fail2ban-testcases',
+		'bin/fail2ban-testcases'
```

Comment:
why?  it was perfectly ok and I usually prefer trailing comma to minimize next diff happen we need to add a new entry

in general -- better to retain only changes related to introduced new functionality in such PRs